### PR TITLE
CEM-1176 Add Validation to MaskedInput

### PR DESCRIPTION
**Pull request by Ampatte2**
MaskedInput had an issue that min and max weren't being implemented on the component, and errors were not given for being outside of the min max.

MaskedInput now has validation and error messaged based on if the value is LT or GT the min and max values respectively.

MaskedInput

### CODE REVIEW

File: stories/Containers/List.stories.tsx [hunk 1]
```diff
@@ -57,8 +65,8 @@ export default {
 } as Meta;
 
 export const Basic: Story<ListProps> = (args) => {
-    const [isOpen, setIsOpen] = useState(true);
-
+    const [{ isOpen }, updateArgs] = useArgs();
```
Comment (ralph-dev) [inline stories/Containers/List.stories.tsx:68]: I like this a lot.

**Review by ralph-dev — APPROVED** [review]
LGTM, just note that 100 is for percentage and 100 00 is value becuase of the /100. I don't think anything needs to change just want to highlight this for Jun

**Review by roggc — COMMENTED** [review]
lgtm.

File: stories/Containers/List.stories.tsx (hunk 1 above)
Comment (Ampatte2) [reply stories/Containers/List.stories.tsx:68]: There was a solution!

Comment (ralph-dev) [reply stories/Containers/List.stories.tsx:68]: @williancozzi He found the solution

**Pull request merged**